src/bug-filing/null-issue-filing-service/null-issue-filing-service.tsx

```diff
@@ -16,5 +16,5 @@ export const NullIssueFilingService: BugFilingService = {
     renderSettingsForm,
     buildStoreData: () => null,
     isSettingsValid: () => false,
-    createBugFilingUrl: () => null,
+    issueFilingUrlProvider: () => null,
```

Comment:
method names should be verb

Reply:
depends of what this method does, may be call it getIssueFilingUrlProvider or createIssueFilingUrlProvider

Reply:
it doesn't return a provider... it provides the url, hence the wording